javatests/google/registry/export/ExportConstantsTest.java

```diff
@@ -17,60 +17,72 @@
 import static com.google.common.io.Resources.getResource;
 import static com.google.common.truth.Truth.assertThat;
 import static com.google.common.truth.Truth.assertWithMessage;
+import static google.registry.model.EntityClasses.CLASS_TO_KIND_FUNCTION;
+import static google.registry.util.TypeUtils.hasAnnotation;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 import com.google.common.base.Function;
 import com.google.common.base.Joiner;
 import com.google.common.base.Splitter;
 import com.google.common.collect.FluentIterable;
 import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Ordering;
 import com.google.common.io.Resources;
 import com.google.re2j.Pattern;
 import com.googlecode.objectify.annotation.Entity;
-import google.registry.model.ImmutableObject;
-import java.net.URL;
-import java.util.List;
-import javax.annotation.Nullable;
+import google.registry.model.EntityClasses;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
+import java.net.URL;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
 /** Unit tests for {@link ExportConstants}. */
 @RunWith(JUnit4.class)
 public class ExportConstantsTest {
 
   private static final String GOLDEN_BACKUP_KINDS_FILENAME = "backup_kinds.txt";
 
-  private static final String UPDATE_INSTRUCTIONS_TEMPLATE = Joiner.on('\n').join(
-      "",
-      "---------------------------------------------------------------------------------",
-      "Your changes affect the list of backed-up kinds in the golden file:",
-      "  %s",
-      "If these changes are desired, update the golden file with the following contents:",
-      "=================================================================================",
-      "%s",
-      "=================================================================================",
-      "");
+  private static final String UPDATE_INSTRUCTIONS_TEMPLATE =
```

Comment:
Did you run code format on this file?  I'm seeing lots of spurious diffs.  Can you scope down the PR to just making the changes that you want to make?  Thanks.